ivy/functional/backends/paddle/experimental/creation.py

```diff
@@ -203,3 +206,46 @@ def tril_indices(
     return tuple(to_device(
         paddle.tril_indices(n_rows, col=n_cols, offset=k, dtype="int64"), device
     ))
+
+
+@with_unsupported_dtypes(
+    {"1.11.0 and below": ("bfloat16", "complex64", "complex128")},
+    backend_version
+)
```

Comment:
I think this is related to torch, please fix this to paddle, note that paddle doesn't support bfloat16 or uint16 at all